content/api/vision.md

```diff
@@ -72,3 +72,14 @@ There are three main marker types in the game `WALL`, `TOKEN` and `TOKEN_ZONE_x`
   - `TOKEN_ZONE_1` - Markers in the 1st zone with IDs from 49 - 53
   - `TOKEN_ZONE_2` - Markers in the 2nd zone with IDs from 54 - 58
   - `TOKEN_ZONE_3` - Markers in the 3rd zone with IDs from 59 - 63
+
+## Marker PDFs
+
+You can download PDF files of all markers:
+
+- [Large markers][large-marker-pdf], (should be printed on A3)
```

Comment:
I suggest we actually tell them what tokens are in which file (by type not ID)

Reply:
I agree, would be good to say "wall" / "column" / "token" markers here, as that's how we refer to them elsewhere in the docs.

The commas in this two lines also seem unnecessary to me.